tools/cbmc/proofs/DNS/DNSlookup/cbmc-batch.yaml

```diff
@@ -0,0 +1,4 @@
+jobos: ubuntu16
+cbmcflags: '=--object-bits;7;--32;--bounds-check;--pointer-check;--unwind;1;--unwindset;prvProcessDNSCache.0:5;--unwinding-assertions='
```

Comment:
Shouldn't `unwindset` also include `strcmp.0:{HOSTNAME_UNWIND}`, as it does in `Makefile.json`?

@markrtuttle could we please make a policy decision about whether to commit `cbmc-batch.yaml` files? I can't remember if we decided either way. I don't think this is the first time that we haven't updated a `cbmc-batch.yaml` file after making changes to the source.

I advocate that we do not commit these files, to prevent this kind of mistake. I've been trying to look out for them, and I caught this one, but these semicolon-delimited lists are difficult to read and compare with `Makefile.json`. It's entirely possible that I've let something slip in a different review, this worries me.

If you would like to continue committing these files, then I advocate adding a "linter" exclusively for our proofs. I could write that. One thing it would check is that `cbmc-batch.yaml` and `Makefile.json` match. If the linter fails then it blocks the PR.